doc/MAINLINE_MODS.md

```diff
@@ -0,0 +1,57 @@
+
+## Why have mainlined mods?
+
+The benefits of having mods mainlined include:
+
+*  Visibility to a large number of users and potential contributors
+*  The ability to use the CleverRaven issue tracker to report problems, as well as manage pull requests for adding to or fixing mods in the repository
+*  Mods in mainline that demonstrate usage of a game feature which exists only for mods (i.e. not used by the core DDA game itself) help ensure that feature stays present in the game.
+
+There are however things that might be expected but are not guaranteed:
+
+*  Guarantee of maintenance of content -- core devs are mostly focused on the main game itself; although some might pick up mods as a side task, people who continue working on the core game tend to like the feel of the core game.
+*  Guarantee of maintenance of used features -- over the course of development it may become impossible to keep a feature, due to performance needs or maintenance burden.
+*  Guarantee of sole authorship -- once in the repository, the mod is considered community content. While the developers might defer to an active maintainer of a mod, if they become inactive anyone else can step forward.
+
+## What kind of mods can be in the repository as mainlined mods?
+
+There are three primary categories of mods:
+
+*  Content mods, which provide some kind of distinct experience from the core Dark Days Ahead game. This could be a change to lore, a change to a specific part of gameplay.
+*  User Experience (UX) mods, which alter the look and feel of the game interface itself.
+*  Development mods, which aren't "mods" in the typical sense but are instead there to ease the transition between "incomplete feature" and "incomplete feature", when a feature in the core game is sufficiently incomplete that the developers believe it needs to be optional to minimize disruption to players.
+
+## What is necessary for a mod to be included in the repository?
+
+The most crucial criteria for a mod to be in the CleverRaven repository is that it has someone acting as a maintainer. This ensures that there is someone who is keeping an eye out for possible problems with the mod, and helping steer its development so that it continues to develop in accordance with its design purpose.
+
+Further, there are additional criteria:
+
+*  Content mods need to be providing some kind of curated experience. Either they introduce a set of locations, encounters, equipment, or progression mechanisms that do not fit the style of the core game, or they in some fashion modify the game according to some well-defined concept.
+*  *  Mods which do not qualify as curated experiences: Grab bag mods with no defined purpose (i.e. adding a bunch of randomly guns), "settings" mods that just turn off a possibly undesired but working feature (acid ants).
```

Comment:
I don't have a problem with this in theory, but it feels a little silly to do this when there's a whole game-supported category called "blacklist" for mods like this.

Reply:
I think it's probably reasonable to leave blacklist as a category for two reasons:
1. it's going to be common in community added mods, and 
2. we're still going to need it for the occasional feature that's in development or has problems on low-end systems. Thinking of fungus and railways here.

Reply:
Don't forget we still have Classic Zombies; that isn't being obsoleted because it is providing a distinct experience rather than 'just' a toggle.

Reply:
Don't forget the classic zombies mod; it's still staying in the repository.